### DISCUSSION TITLE
[Feature Request] Add lower limit / minimum to Assets fields

### DISCUSSION THREAD

**Discussion by proimage** [craftcms/cms#8621] [Ideas] [answered]
The Assets fieldtype has a "Limit" field, which defines the maximum number of assets that can be uploaded to that field. What it's missing is a "Minimum" field of some kind, to ensure that the user uploads _at least X assets_.

(Incredibly, I was unable to find this as an existing issue/feature request; if I missed it, many pardons!)

**Comment by ghost** [comment]
Yes, please.

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4! (PR: #10402)
